docs/docs/0.5.0/policies/health-check.md

```diff
@@ -1,51 +1,31 @@
 # Health Check
 
-The goal of Health Checks is to minimize the number of failed requests due to temporary unavailability of a target endpoint.
-By applying a Health Check policy you effectively instruct a dataplane to keep track of health statuses for target endpoints.
-Dataplane will never send a request to an endpoint that is considered "unhealthy".
+This policy enables Kuma to keep track of the health of every data plane proxy, with the goal of minimizing the number of failed requests in case a data plane proxy is temporarily unhealthy.
 
-Since pro-active health checking might result in a tangible extra load on your applications,
-Kuma also provides a zero-overhead alternative - ["passive" health checking](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/upstream/outlier).
-In the latter case, a dataplane will be making decisions whether target endpoints are healthy based on "real" requests
-initiated by your application rather than auxiliary requests initiated by the dataplanes itself.
+By creating an `Healthcheck` resource we can instruct a data plane proxy to keep track of the health status for any other data plane proxy. When health-checks are properly configured, a data plane proxy will never send a request to another data plane proxy that is considered unhealthy.
```

Comment:
Probably worth mentioning that unhealthy dataplane might become healthy again. Because otherwise that part "data plane proxy will never send a request to another data plane" sounds a bit confusing